README.md

```
@@ -2,313 +2,1410 @@
+- [Srijan Baniyal](https://srijanbaniyal.com) [Full Stack Developer]
+- [Ste O'Neill](https://steoneill.dev)
+- [Stefan Bohacek](https://fourtonfish.com)
+- [Stefan Topalovic](https://www.stefantopalovic.com)
+- [Stephanie Lin](https://lin-stephanie.github.io)
+- [Stephen Adebanjo](https://myportfoliome.vercel.app)
+- [Stéphane Chan Hiou Kong](https://www.chan-stephane.me) [Software Engineer] ([@Chan-Stephane](https://github.com/chan-stephane))
+- [Subham Saha](https://subhamsaha.in) [Mern Stack & Next.Js Developer]
+- [Subhan Nadeem](https://subhan-dev-portfolio.vercel.app)
+  ([@Satishjhanwer](https://github.com/satishjhanwer))
+- [Subhasish Das](https://subhasish-portfolio.vercel.app)
+- [Sudarshan Rai](https://sudarshanrai.com.np)[Full Stack Developer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://sudarshanrai.com.np)